Tasks/Gradle/gradletask.ts

```diff
@@ -5,7 +5,7 @@ import fs = require('fs');
 import path = require('path');
 
 // Lowercased file names are to lessen the likelihood of xplat issues
-import sqCommon = require('sonarqube-common/sonarqube-common');
+import sqCommon = require('./sonarqube-common/sonarqube-common');
```

Comment:
Consider renaming this directory to "SonarQube" and moving it into CodeAnalysis. Also, there is no need for all the file names to be prefixed with sonarqube-